tfx/dsl/component/experimental/annotations.py

```diff
@@ -183,10 +191,6 @@ class BeamComponentParameter(_PipelineTypeGeneric):
   pass
 
 
-# TODO(ccy): potentially make this compatible `typing.TypedDict` in
-# Python 3.8, to allow for component return value type checking.
-class OutputDict:
-  """Decorator declaring component executor function outputs."""
-
-  def __init__(self, **kwargs):
-    self.kwargs = kwargs
+class OutputDict(TypedDict):
```

Comment:
Does this break anything because TypedDict expects the name as the first argument?